### ISSUE TITLE
`azwi serviceaccount` support for managed identity

### ISSUE THREAD

**Issue by aramase** [Azure/azure-workload-identity#414]
`azwi serviceaccount` subcommands only support creation/deletion of Azure AD Application and FIC for the same. We need to extend it to support creating/deleting managed identity/FIC for managed identity in addition to Azure AD Applications.

dependent on #325

**Comment by ohorvath** [comment]
If I' understand correctly with the current version there is no way to annotate the Kubernetes service account with a user assigned managed identity? Our product has been designed to use only managed identities for everything from AKS cluster creation to pod identities/Azure resource access. Without this feature there is no way for us to move away from the current ADD pod identity solution. Do you think this can be included in the GA release?

**Comment by aramase** [comment]
> If I' understand correctly with the current version there is no way to annotate the Kubernetes service account with a user assigned managed identity? Our product has been designed to use only managed identities for everything from AKS cluster creation to pod identities/Azure resource access. Without this feature there is no way for us to move away from the current ADD pod identity solution. Do you think this can be included in the GA release?

@ohorvath Azure AD currently doesn't support workload identity federation with managed identity. This is work in progress and is being tracked in this [issue](https://github.com/Azure/azure-workload-identity/issues/325). Here is some info in our faq: https://azure.github.io/azure-workload-identity/docs/faq.html#why-is-managed-identity-not-supported-by-azure-workload-identity.

For GA release, in addition to other features, we are also tracking workload identity federation for managed identity support in AAD.

**Comment by ohorvath** [comment]
Thanks, sounds good.

**Comment by raorugan** [comment]
@aramase  - by when do you plan to GA launch the workload identity federation for managed identity support in AAD? Any timeline that can be shared. This is critical for our service launch. I can provide more details on the service if needed.

**Comment by tshaiman** [comment]
@aramase  : you sent a link to a closed item. can you reflect on the status here ?

**Comment by aramase** [comment]
> @aramase : you sent a link to a closed item. can you reflect on the status here ?

@tshaiman which item is closed? This issue is open. I closed #808 as it's a duplicate of this.
